src/pipeline_v3/tests/integration/test_e2e_integration.py

```
@@ -99,10 +109,17 @@ async def test_document_ingestion(
 
                 # Verify the document was processed
-                assert result is not None
-                assert "doc_id" in result
-                assert result["status"] == "success"
-                assert result["action"] == "indexed"
+                assert result is not None, f"Result is None for {doc_path.name}"
+                assert "doc_id" in result, f"No doc_id in result: {result}"
+                # Print result for debugging in CI
+                print(f"Processing result: {result}")
+                if result.get("status") != "success":
+                    print(f"ERROR: Processing failed!")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)